### PR TITLE
Tests/add dummy app

### PR DESCRIPTION
**Pull request by Ricoch**
Add dummy app in order to test gem functionalities

### CODE REVIEW

File: spec/dummy/app/assets/stylesheets/active_admin.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// SASS variable overrides must be declared before loading up Active Admin's styles.
```
Comment (santib) [inline spec/dummy/app/assets/stylesheets/active_admin.css]: are these valid css comments? look weird in Github and in https://developer.mozilla.org/en-US/docs/Web/CSS/Comments says that comments are like `/* ... */`

**Review by santib — APPROVED** [review]
👏

File: spec/dummy/app/admin/dashboard.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  content title: proc { I18n.t('active_admin.dashboard') } do
+    div class: 'blank_slate_container', id: 'dashboard_default_message' do
+      span class: 'blank_slate' do
+        span I18n.t('active_admin.dashboard_welcome.welcome')
+        small I18n.t('active_admin.dashboard_welcome.call_to_action')
+      end
+    end
+
+    # Here is an example of a simple dashboard with columns and panels.
```
Comment (jpascual1994) [inline spec/dummy/app/admin/dashboard.rb]: Remove comments

**Review by jpascual1994 — APPROVED** [review]
Great job 🚀

File: spec/dummy/app/admin/dashboard.rb (hunk 2 above)
Comment (Ricoch) [reply spec/dummy/app/admin/dashboard.rb]: We probably should consider removing it from the rails_api_base as well, WDYT?
https://github.com/rootstrap/rails_api_base/blob/master/app/admin/dashboard.rb

File: spec/dummy/app/assets/stylesheets/active_admin.css (hunk 1 above)
Comment (Ricoch) [reply spec/dummy/app/assets/stylesheets/active_admin.css]: Will check! You are probably right though

**Review by santib — APPROVED** [review]
👏

**Pull request merged**